colbert/__init__.py

```diff
@@ -1,5 +1,6 @@
 from .trainer import Trainer
 from .indexer import Indexer
 from .searcher import Searcher
+from .incremental_indexer import PassageRemover, PassageAppender
```

Comment:
Should this be updated to the new ones?

Reply:
Yeah, IndexUpdater is imported in line 5, I'll delete line 4 as it's no longer needed.